website/content/docs/audit/index.mdx

```diff
@@ -74,20 +74,20 @@ The existing logs that it did store are untouched.
 
 ## Blocked Audit Devices
 
-If there are any audit devices enabled, Vault requires that at least
-one be able to persist the log before completing a Vault request.
+If the audit device request becomes blocked for an indefinite period of time,
```

Comment:
I wonder if we should be a little more explicit about what we mean by "blocking"?  I just worry that some might interpret e.g. a full disk as a cause that that they should worry about here.

Also, though a lesser concern, "audit device request" is a bit of a weird phrasing.

How about something more like:

We can distinguish between two kinds of audit device failure.  A blocking failure is one where an attempt to write to the audit device never completes.  This is unlikely with a local disk device, but it could happen with a network-based audit device.

With multiple audit devices, if any of them fail in a non-blocking fashion, Vault requests can still complete successfully provided at least one audit device successfully writes the audit record.  But if any of them fail in a blocking fashion, Vault requests will hang until the blocking is resolved.  In other words, Vault **will not** complete any requests until the blocked audit device can write.